php/Middleware/AuthMiddleware.php

```diff
@@ -28,6 +28,15 @@ public function __invoke(Request $request, RequestHandler $handler): Response
 
 		if ($routeName === 'root')
 		{
+			if (GROCY_MODE === 'dev' || GROCY_MODE === 'demo' || GROCY_MODE === 'prerelease' || GROCY_IS_EMBEDDED_INSTALL || GROCY_DISABLE_AUTH)
+			{
+				$sessionService = SessionService::getInstance();
+				$user = $sessionService->GetDefaultUser();
```

Comment:
possibly this

Reply:
Yes https://github.com/mistressofjellyfish/not-grocy/pull/25/checks?check_run_id=2980622271#step:8:144 line 34 it is

Reply:
I think we have a winner here.